src/server/getters/get-dispute-info.ts

```diff
@@ -15,6 +16,8 @@ interface DisputesResult {
   markets: Array<MarketsRowWithCreationTime>;
   completedStakes: Array<StakeRow>;
   activeCrowdsourcer: Array<ActiveCrowdsourcer>;
+  accountStakeComplete: Array<any>;
+  accountStakeIncomplete: Array<any>;
   payouts: Array<PayoutRow>;
```

Comment:
can <any> be <string> or nullable string or is it literally any value will work?

Reply:
You're referencing the `Array<any>` ? I didn't mean to leave those in there, I'll fix.